CMakeLists.txt

```diff
@@ -567,7 +567,7 @@ endif (EXISTS "${PROJECT_SOURCE_DIR}/.svn")
 
 # N.B.: when updating this, update the default version in ci-package.yml.
 # We should find a way to share (xref i#1565).
-set(VERSION_NUMBER_DEFAULT "9.0.${VERSION_NUMBER_PATCHLEVEL}")
+set(VERSION_NUMBER_DEFAULT "9.90.${VERSION_NUMBER_PATCHLEVEL}")
```

Comment:
Only a minor level bump?

Reply:
https://dynamorio.org/page_new_release.html#autotoc_md227
"if we break backward compatibility during development, we expect to increment the major number on the next release and we immediately change the minor number to 90. "

Reply:
Okay.